proposals/0000-float16.md

```diff
@@ -0,0 +1,134 @@
+# Float16
+
+* Proposal: [SE-0000](0000-float16.md)
+* Author: [Stephen Canon](https://github.com/stephentyrone)
+* Review Manager: TBD
+* Pitch thread: [Add Float16](https://forums.swift.org/t/add-float16/33019)
+* Implementation: [Implement Float16](https://github.com/apple/swift/pull/21738)
+* Status: **Awaiting review**
+
+## Introduction
+
+Introduce the `Float16` type conforming to the `BinaryFloatingPoint` and `SIMDScalar`
+protocols, binding the IEEE 754 *binary16* format (aka *float16*, *half-precision*, or *half*),
+and bridged by the compiler to the C `_Float16` type.
+
+* Old pitch thread: [Add `Float16`](https://forums.swift.org/t/add-float16/19370).
+
+## Motivation
+
+The last decade has seen a dramatic increase in the use of floating-point types smaller
+than (32-bit) `Float`. The most widely implemented is `Float16`, which is used
```

Comment:
It might be clearer to use `Float32` and `Float64` throughout.

```suggestion
than `Float32`. The most widely implemented is `Float16`, which is used
```

Reply:
I think most people reviewing the proposal are more familiar with the names `Float` and `Double` (indeed, several people were surprised that the other names exist).